### PR TITLE
Update webdriver and scalafmt rules

### PR DESCRIPTION
**Pull request by RawToast**
Currently builds will fail on Jenkins due to this dependency being out of date

Also updated the formatting rules to match the main project and handled the bobby warning

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -11,7 +11,7 @@ object Dependencies {
     "io.cucumber"          % "cucumber-junit"    % "7.6.0"    % Test,
     "junit"                % "junit"             % "4.13.2"   % Test,
     "com.novocode"         % "junit-interface"   % "0.11"     % Test,
-    "uk.gov.hmrc"         %% "webdriver-factory" % "0.37.0"   % Test
+    "uk.gov.hmrc"         %% "webdriver-factory" % "0.41.0"   % Test
```
Comment (RawToast) [inline project/Dependencies.scala:14]: Without updating this, the tests will not run on CI

File: project/build.properties [hunk 2]
```diff
@@ -1 +1 @@
-sbt.version=1.6.2
+sbt.version=1.7.2
```
Comment (RawToast) [inline project/build.properties:1]: Fixes bobby warning

File: src/test/resources/application.conf [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 HM Revenue & Customs
```
Comment (RawToast) [inline src/test/resources/application.conf:1]: From here on, the changes will be just copyright notices & any minor formatting changes

File: build.sbt [hunk 4]
```diff
@@ -1,11 +1,13 @@
 lazy val testSuite = (project in file("."))
-  .disablePlugins(JUnitXmlReportPlugin) //Required to prevent https://github.com/scalatest/scalatest/issues/1427
+  .disablePlugins(
+    JUnitXmlReportPlugin
```
Comment (C-Thomp) [inline build.sbt:3]: random /n?

**Review by C-Thomp — APPROVED** [review]
👍 
thanks for the comment about copyright change haha

File: build.sbt (hunk 4 above)
Comment (RawToast) [reply build.sbt:3]: interestingly, there's a scalafmtSbtAll command that formats this file 😮

**Pull request merged**